c3/optimizers/c3.py

```diff
@@ -196,7 +197,9 @@ def learn_model(self) -> None:
         except KeyboardInterrupt:
             pass
         with open(self.logdir + "best_point_" + self.logname, "r") as file:
-            best_params = hjson.loads(file.readlines()[1])["params"]
+            best_params = hjson.loads(
```

Comment:
This was updated in #117 to be at par with the structure in the rest of the codebase. So this will require a merge conflict resolution.

Reply:
Made the test files smaller

Reply:
I believe I did this correctly as all tests passed